server/routes/charts.js

```diff
@@ -158,12 +159,41 @@ router.route('/charts/:chart_id')
               return currentSite;
             }, {}))
         )
+
+      const legendquery = [
```

Comment:
nitpick: would prefer camel case

```suggestion
      const legendQuery = [
```

Reply:
[dc64da2](https://github.com/AMP-SCZ/dpdash/pull/99/commits/dc64da28057685b5e1439fa007b7bc9ef54e42c7)